skrub/_table_vectorizer.py

```diff
@@ -191,6 +192,9 @@ class TableVectorizer(TransformerMixin, BaseEstimator):
         similar functionality to what is offered by scikit-learn's
         :class:`~sklearn.compose.ColumnTransformer`.
 
+    drop_null_columns : bool, default=False
```

Comment:
Do we want it to be `True` by default ?

Reply:
That should be discussed with others I think

Reply:
I vote for true by default -- there's nothing we can learn from a completely empty column.

if it is False by default, I think it should be set to True in the `tabular_learner`